lib/format-jsx.js

```diff
@@ -332,7 +332,8 @@ module.exports = {
                 line = line.split( key ).join( parts.join( '\n' ) );
                 line = line.replace( /\s+$/, '' );
 
-                delete expressionContainers[ key ];
+                delete expressionContainers[ key ]
```

Comment:
oh crap, this is a bug on esformatter... forget to mention how to fix this... will fix in master before publishing a new one.